sovrin_client/test/cli/test_tutorial_manual.py

```diff
@@ -274,11 +257,13 @@ def executeGstFlow(name, userCLI, userMap, be, connectedToTest, do, fMap,
                    syncedInviteAcceptedOutWithoutClaims, tMap,
                    transcriptClaimMap)
 
-    # TODO: restart agents to test everything still works fine
-    # faberAgent, acmeAgent, thriftAgent = restartAgents()
+    aliceCLI.looper.runFor(3)
 
-    executeGstFlow("Susan", susanCLI, susanMap, be, connectedToTest, do, fMap,
-                   aMap, jobCertificateClaimMap, newKeyringOut, reqClaimOut,
-                   reqClaimOut1, syncLinkOutWithEndpoint,
-                   syncedInviteAcceptedOutWithoutClaims, tMap,
-                   transcriptClaimMap)
+    # Same flow is executed by different cli
```

Comment:
That is fine from tests perspective, but what about actual scenario, when this code would be in master, what will happen if two users will do the same thing which I mentioned, the second user won't get the same output as mentioned in getting started tutorail.